db/seeds.rb

```diff
@@ -1,6 +1,18 @@
-User.destroy_all
-MonthlyReport.destroy_all
-Tag.destroy_all
+[
+  ActiveAdminActionLog,
+  User,
+  MonthlyReport,
+  Tag,
+  HelpText,
+].map(&:destroy_all)
+
+Dir.glob("#{Rails.root}/db/seeds/*.yml").each do |yaml_filename|
+  klass = File.basename(yaml_filename,".yml").classify.constantize
```

Comment:
__[rubocop]__
* Space missing after comma. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)